src/components/Dashboard.js

```diff
@@ -1,44 +1,83 @@
+      });
+      let formattedData = formatFetchData(historicalDataArray);
+      console.log('formattedData', formattedData);
+      let formattedHistoryData = formatChartData(formattedData);
+      setHistoryData(formattedHistoryData);
+    };
+
+    fetchHistoricalData();
+  }, []);
+
+  useEffect(() => {
+    const fetchCurrentData = async () => {
+      let currentDataArray = [];
+      await Axios.get(
+        'https://api.coingecko.com/api/v3/coins/raptoreum?localization=de&tickers=false&market_data=true&developer_data=false&sparkline=false'
+      ).then((response) => {
+        currentDataArray = response.data;
+      });
+      setCoin(currentDataArray);
+    };
+
+    fetchCurrentData();
+  }, []);
+  console.log('coin', coin);
+  console.log('historyData', historyData);
+
   return (
     <MainDashboard>
       <DashboardTitle>Dashboard</DashboardTitle>
-      <CurrencySelector>
-        <option>Raptoreum</option>
-        <option>Bitcoin</option>
-        <option>Ethereum</option>
-        <option>Matic</option>
-        <option>Cardano</option>
-      </CurrencySelector>
-      <DashboardGraph>
-        <DashboardGraphDummy src={dummy} alt="Value-Graph" />
-      </DashboardGraph>
-      <DashboardTableContainer>
-        <TableIndicator>Preis:</TableIndicator>
-        <TableValue>0,01 USD</TableValue>
-        <TableIndicator>Marktkapazität:</TableIndicator>
-        <TableValue>13.722.000 USD</TableValue>
-        <TableIndicator>24h-Volumen:</TableIndicator>
-        <TableValue>68.868 USD</TableValue>
-        <TableIndicator>Maximalmenge:</TableIndicator>
-        <TableValue>21.000.000.000 RTM</TableValue>
-      </DashboardTableContainer>
-      <NewsTitle>News</NewsTitle>
-      <NewsArticle>
-        <NewsArticleHeadline>2 Millionen Wallets</NewsArticleHeadline>
-        <NewsArticleText>
-          Es existieren mittlerweile zwei Millionen Wallets, die Raptoreum-Coins
-          aufbewahren: ein rasanter Anstieg der Nutzer seit Verfügbarkeitsbeginn
-          des Coins vor 18 Monaten.
-        </NewsArticleText>
-      </NewsArticle>
+      {coin ? <CoinName>{coin.name}</CoinName> : <div>Loading...</div>}
+      {historyData ? (
+        <HistoryChart historyData={historyData} />
+      ) : (
+        <div>Loading...</div>
+      )}
+      {coin ? (
+        <DashboardTableContainer>
+          <TableIndicator>Preis:</TableIndicator>
+          <TableValue>{coin.market_data.current_price.eur} EUR</TableValue>
+          <TableIndicator>Marktkapazität:</TableIndicator>
+          <TableValue>{coin.market_data.market_cap.eur} EUR</TableValue>
+          <TableIndicator>Rang (Marktkapazität):</TableIndicator>
+          <TableValue>{coin.market_data.market_cap_rank}</TableValue>
+          <TableIndicator>Maximalmenge:</TableIndicator>
+          <TableValue>{coin.market_data.max_supply} RTM</TableValue>
+        </DashboardTableContainer>
+      ) : (
+        <div>Loading...</div>
+      )}
```

Comment:
It is a bit unusual to see three `Loading` texts at the screen. Maybe rework the conditional rendering to this: Or something like that 😅 
```suggestion
      {coin.length === 0 || historyData === 0  && <div>Loading...</div>}
      {historyData.length > 0 && <HistoryChart historyData={historyData} />}
      {coin.length > 0 && (
        <DashboardTableContainer>
          <TableIndicator>Preis:</TableIndicator>
          <TableValue>{coin.market_data.current_price.eur} EUR</TableValue>
          <TableIndicator>Marktkapazität:</TableIndicator>
          <TableValue>{coin.market_data.market_cap.eur} EUR</TableValue>
          <TableIndicator>Rang (Marktkapazität):</TableIndicator>
          <TableValue>{coin.market_data.market_cap_rank}</TableValue>
          <TableIndicator>Maximalmenge:</TableIndicator>
          <TableValue>{coin.market_data.max_supply} RTM</TableValue>
        </DashboardTableContainer>
      )}
```